test/functional/rpc_psbt.py

```
@@ -499,7 +502,245 @@
         assert not signed['complete']
         signed = self.nodes[0].walletprocesspsbt(signed['psbt'])
         assert signed['complete']
-        self.nodes[0].finalizepsbt(signed['psbt'])
+        final = self.nodes[0].finalizepsbt(signed['psbt'], False)
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)